src/arch/mod.rs

```diff
@@ -344,4 +350,18 @@ mod tests {
         let ty = struct_for_types(vec![Pointer(Box::new(Int(true))), Int(true)]);
         assert_eq!(ty.alignof(), Ok(8));
     }
+
+    proptest! {
+        #[test]
+        fn proptest_sizeof_alignof_same_variant(t in arb_type()) {
```

Comment:
This can just be `sizeof_alignof` since we're testing things besides the discriminant